File: api/Representations/DTO/VesselTypeDto.cs
public class VesselTypeDto
{
    public Guid Id { get; set; }
    public string Name{ get; set; }
    public string Description{ get; set; }
    public int MaxNumberOfRows{ get; set; }
    public int MaxNumberOfBays{ get; set; }
    public int MaxNumberOfTiers{ get; set; }
}

File: FrontEnd/src/model/dto/VesselTypeDto.ts
import type { PhysicalCharacteristics } from "../values/PhysicalCharacteristics";

export interface VesselTypeDto {
    name: string;
    description: string;
    maxNumberOfRows: number;
    maxNumberOfBays: number;
    maxNumberOfTiers: number;
    capacity?: number;
    physicalCharacteristics: PhysicalCharacteristics;
}